WordPress/Classes/ViewRelated/Jetpack/Install/JetpackInstallPluginHelper.swift

```diff
@@ -0,0 +1,46 @@
+@objcMembers
+class JetpackInstallPluginHelper: NSObject {
+
+    private static var cardHiddenSites: [String] {
+        get {
+            guard let sites = UserPersistentStoreFactory.instance().array(forKey: Constants.cardHiddenSitesKey) as? [String] else {
+                return [String]()
+            }
+            return sites
+        }
+        set {
+            UserPersistentStoreFactory.instance().set(newValue, forKey: Constants.cardHiddenSitesKey)
+        }
+    }
+
+    static func shouldShow(for blog: Blog?) -> Bool {
+        guard let blog, blog.account != nil else {
+            return false
+        }
+        let isFeatureEnabled = FeatureFlag.jetpackIndividualPluginSupport.enabled
+        return isFeatureEnabled && blog.jetpackIsConnectedWithoutFullPlugin
```

Comment:
Noting that we'll need to add the `isJetpack` check on the `FeatureFlag` later on.